### DISCUSSION TITLE
How to use desperate 4 candles?

### DISCUSSION THREAD

**Discussion by n163n7** [thatskymod/Sky-CotL-Scripts#135] [Q&A] [answered]
Can somebody explain how desperate 4 candles works? i will really appreciate it, thanks!

**Comment by thatskymod** [comment, marked as answer]
Excellent question - let's explore in detail!

Normally, D4C would be a mod that converted yellow seasonal candles into white regular ones. This was particularly useful in Sky Beta, a testing flavor of the game where in-app purchases are free. In this version, you could 'purchase' unlimited seasonal candle packs, then convert them into regular ones. This was a workaround for players looking to amass a large collection of white candles without the usual time investment.

However, there appears to be a current issue with D4C, as indicated by several messages: [message](https://t.me/skyhackerschat/63421), [message](https://t.me/dukunlangit/5/28087), and [message](https://t.me/dukunlangit/5/28297), among others. So while there are instructions included in libD4C, it seems that even if you follow them properly, the mod won't help you due to recent compatibility issues.

**Reply by n163n7** [reply]
Thanks a lot for the answer!
I'll patiently wait for it to get fixed again.

**Reply by thatskymod** [reply]
Hello,

You’re welcome to try the updated [libD4C_002x.so](https://github.com/thatskymod/Sky-CotL-Scripts/blob/f6866bbf5dbdc45e2cf941b33263c3fbde778a83/Canvas/Libs/libD4C_002x.so). This version now supports both Live & Beta and addresses bugs on Huawei devices. Note that each season can only be converted once. All seasonal currency is converted during the process, including hearts, into regular currency. Make sure you understand the implications before use, as this action is irreversible.
